NAMESPACE

```diff
@@ -90,16 +87,12 @@ S3method(within, data.table)
 S3method(is.na, data.table)
 S3method(format, data.table)
 S3method(Ops, data.table)
-
 S3method(anyDuplicated, data.table)
-
-export(split.data.table)
```

Comment:
is this really required? this was exported in 1.11.6 #2920 as per user request in a comment somewhere. As it is very recent it won't break much code, but NEWS entry would be useful.

Reply:
Added a news item to 1.11.8 removing the export:  #3083